docs/frameworks/symfony.md

```diff
@@ -56,7 +58,11 @@ functions:
 
 We still have a few modifications to do on the application to make it compatible with AWS Lambda.
 
-Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and the logs are stored in the `src/Kernel.php` file. This is done by adding 2 new methods to the `Kernel` class:
+Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache and the logs are stored in the `src/Kernel.php` file. This is done by adding 3 new methods to the `Kernel` class. 
```

Comment:
Just say 2 new methods.

Reply:
But it's actually 3 right? (it includes the "getBuildDir" method)

Reply:
Oh nevermind I finally got what you were trying to say:

- change 2 methods
- **if** you want optimization, do this extra thing

Makes sense, +1 on your suggested changes